pkg/steps/cluster_claim.go

```diff
@@ -184,13 +196,23 @@ func mutate(secret *corev1.Secret, name, namespace string) (*corev1.Secret, erro
 	}, nil
 }
 
-func ClusterClaimStep(as string, clusterClaim *api.ClusterClaim, hiveClient ctrlruntimeclient.Client, client loggingclient.LoggingClient, jobSpec *api.JobSpec) api.Step {
+func releaseCluster(ctx context.Context, hiveClient ctrlruntimeclient.Client, clusterClaim *hivev1.ClusterClaim) error {
+	logrus.Info("Deleting cluster claim.")
```

Comment:
```suggestion
	logrus.withField("clusterClaim.Namespace",clusterClaim.Namespace).withField("clusterClaim.Name",clusterClaim.Name).Debug("Deleting cluster claim.")
```

Otherwise it would look the same as 

https://github.com/openshift/ci-tools/pull/1905/files#diff-67951af79cb529c491eaa6d7fca53e175eb141fbc9df0af3cd74c86eacf9377bR72